spec/lib/pact_broker/ui/controllers/index_spec.rb

```diff
@@ -85,6 +85,56 @@ module Controllers
                 get "/", { tags: "prod" }
               end
             end
+
+            context "when parameter consumer_name presents" do
+              context "when it is blank" do
+                it "ignores it" do
+                  expect(PactBroker::Index::Service).not_to receive(:find_index_items).with(hash_including(:consumer_name))
+                  get "/", { consumer_name: "" }
+                end
+              end
+
+              context "when it is NOT blank and exist" do
```

Comment:
I would prefer not having to many level of indentation, hence the `and` in the
description. Please let me know if you think otherwise